marketstack/src/endpoint/eod.ts

```diff
@@ -0,0 +1,48 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { AdapterRequest, Config } from '@chainlink/types'
+import { DEFAULT_INTERVAL, DEFAULT_LIMIT } from '../config'
+import { util } from '@chainlink/ea-bootstrap'
+
+export const Name = 'eod'
+
+const customError = (data: any) => data.Response === 'Error'
+
+const customParams = {
+  base: ['base', 'from', 'coin'],
+  interval: false,
+  limit: false,
+}
+
+export const execute = async (config: Config, request: AdapterRequest) => {
+  const validator = new Validator(request, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const symbols = validator.validated.data.base.toUpperCase()
```

Comment:
`symbol`?

Reply:
It is the parameter on marketstack side.